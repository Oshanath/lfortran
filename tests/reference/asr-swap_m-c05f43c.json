{
    "basename": "asr-swap_m-c05f43c",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../examples/swap_m.f90",
    "infile_hash": "8a2d534cc49e105ca27668ef3a14ec4e1952dcbd7657d0bf28550d68",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-swap_m-c05f43c.stdout",
    "stdout_hash": "0303db2ad6fd3f9b3ff5d9ef345cbc4801e46791043a5651c130067d",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}