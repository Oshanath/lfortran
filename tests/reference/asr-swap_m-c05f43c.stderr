BFD: DWARF error: section .debug_info is larger than its filesize! (0x93ef57 vs 0x530ea0)
BFD: DWARF error: section .debug_info is larger than its filesize! (0x93ef57 vs 0x530ea0)
Traceback (most recent call last):
[2m  Binary file "[0m[1m[35m$DIR/src/bin/lfortran[39m[0m[2m", in _start()[0m
[2m  Binary file "[0m[1m[35m/lib/x86_64-linux-gnu/libc.so.6[39m[0m[2m", in __libc_start_main()[0m
[2m  File "[0m[1m[35m$DIR/src/bin/lfortran.cpp[39m[0m[2m", line 1628, in ??[0m
    return emit_asr(arg_file, lfortran_pass_manager,
[2m  File "[0m[1m[35m$DIR/src/bin/lfortran.cpp[39m[0m[2m", line 551, in ??[0m
    r = fe.get_asr2(input, lm, diagnostics);
[2m  File "[0m[1m[35m$DIR/src/lfortran/fortran_evaluator.cpp[39m[0m[2m", line 225, in LFortran::FortranEvaluator::get_asr2(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, LFortran::LocationManager&, LFortran::diag::Diagnostics&)[0m
    Result<ASR::TranslationUnit_t*> res2 = get_asr3(*ast, diagnostics);
[2m  File "[0m[1m[35m$DIR/src/lfortran/fortran_evaluator.cpp[39m[0m[2m", line 247, in LFortran::FortranEvaluator::get_asr3(LFortran::AST::TranslationUnit_t&, LFortran::diag::Diagnostics&)[0m
    compiler_options.symtab_only);
[2m  File "[0m[1m[35m$DIR/src/lfortran/semantics/ast_to_asr.cpp[39m[0m[2m", line 36, in LFortran::ast_to_asr(Allocator&, LFortran::AST::TranslationUnit_t&, LFortran::diag::Diagnostics&, LFortran::SymbolTable*, bool)[0m
    auto res = symbol_table_visitor(al, ast, diagnostics, symbol_table);
[2m  File "[0m[1m[35m$DIR/src/lfortran/semantics/ast_symboltable_visitor.cpp[39m[0m[2m", line 1317, in LFortran::symbol_table_visitor(Allocator&, LFortran::AST::TranslationUnit_t&, LFortran::diag::Diagnostics&, LFortran::SymbolTable*)[0m
    Error error;
[2m  File "[0m[1m[35m$DIR/src/lfortran/semantics/ast_symboltable_visitor.cpp[39m[0m[2m", line 121, in LFortran::SymbolTableVisitor::visit_TranslationUnit(LFortran::AST::TranslationUnit_t const&)[0m
    visit_ast(*x.m_items[i]);
[2m  File "[0m[1m[35m$DIR/src/lfortran/ast.h[39m[0m[2m", line 4622, in LFortran::AST::BaseVisitor<LFortran::SymbolTableVisitor>::visit_ast(LFortran::AST::ast_t const&)[0m
    void visit_ast(const ast_t &b) { visit_ast_t(b, self()); }
[2m  File "[0m[1m[35m$DIR/src/lfortran/ast.h[39m[0m[2m", line 4581, in ??[0m
    case astType::mod: { v.visit_mod((const mod_t &)x); return; }
[2m  File "[0m[1m[35m$DIR/src/lfortran/ast.h[39m[0m[2m", line 4625, in LFortran::AST::BaseVisitor<LFortran::SymbolTableVisitor>::visit_mod(LFortran::AST::mod_t const&)[0m
    void visit_mod(const mod_t &b) { visit_mod_t(b, self()); }
[2m  File "[0m[1m[35m$DIR/src/lfortran/ast.h[39m[0m[2m", line 4233, in ??[0m
    case modType::Module: { v.visit_Module((const Module_t &)x); return; }
[2m  File "[0m[1m[35m$DIR/src/lfortran/semantics/ast_symboltable_visitor.cpp[39m[0m[2m", line 226, in LFortran::SymbolTableVisitor::visit_Module(LFortran::AST::Module_t const&)[0m
    visit_ModuleSubmoduleCommon<AST::Module_t, ASR::Module_t>(x);
[2m  File "[0m[1m[35m$DIR/src/lfortran/semantics/ast_symboltable_visitor.cpp[39m[0m[2m", line 199, in void LFortran::SymbolTableVisitor::visit_ModuleSubmoduleCommon<LFortran::AST::Module_t, LFortran::ASR::Module_t>(LFortran::AST::Module_t const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >)[0m
    visit_unit_decl2(*x.m_decl[i]);
[2m  File "[0m[1m[35m$DIR/src/lfortran/ast.h[39m[0m[2m", line 4636, in LFortran::AST::BaseVisitor<LFortran::SymbolTableVisitor>::visit_unit_decl2(LFortran::AST::unit_decl2_t const&)[0m
    void visit_unit_decl2(const unit_decl2_t &b) { visit_unit_decl2_t(b, self()); }
[2m  File "[0m[1m[35m$DIR/src/lfortran/ast.h[39m[0m[2m", line 4265, in ??[0m
    case unit_decl2Type::Template: { v.visit_Template((const Template_t &)x); return; }
[2m  File "[0m[1m[35m$DIR/src/lfortran/semantics/ast_symboltable_visitor.cpp[39m[0m[2m", line 1285, in LFortran::SymbolTableVisitor::visit_Template(LFortran::AST::Template_t const&)[0m
[2m  File "[0m[1m[35m$DIR/src/lfortran/ast.h[39m[0m[2m", line 4630, in LFortran::AST::BaseVisitor<LFortran::SymbolTableVisitor>::visit_program_unit(LFortran::AST::program_unit_t const&)[0m
    void visit_program_unit(const program_unit_t &b) { visit_program_unit_t(b, self()); }
[2m  File "[0m[1m[35m$DIR/src/lfortran/ast.h[39m[0m[2m", line 4244, in ??[0m
    case program_unitType::Subroutine: { v.visit_Subroutine((const Subroutine_t &)x); return; }
[2m  File "[0m[1m[35m$DIR/src/lfortran/semantics/ast_symboltable_visitor.cpp[39m[0m[2m", line 280, in LFortran::SymbolTableVisitor::visit_Subroutine(LFortran::AST::Subroutine_t const&)[0m
    visit_unit_decl2(*x.m_decl[i]);
[2m  File "[0m[1m[35m$DIR/src/lfortran/ast.h[39m[0m[2m", line 4636, in LFortran::AST::BaseVisitor<LFortran::SymbolTableVisitor>::visit_unit_decl2(LFortran::AST::unit_decl2_t const&)[0m
    void visit_unit_decl2(const unit_decl2_t &b) { visit_unit_decl2_t(b, self()); }
[2m  File "[0m[1m[35m$DIR/src/lfortran/ast.h[39m[0m[2m", line 4262, in ??[0m
    case unit_decl2Type::Declaration: { v.visit_Declaration((const Declaration_t &)x); return; }
[2m  File "[0m[1m[35m$DIR/src/lfortran/semantics/ast_symboltable_visitor.cpp[39m[0m[2m", line 572, in LFortran::SymbolTableVisitor::visit_Declaration(LFortran::AST::Declaration_t const&)[0m
    visit_DeclarationUtil(x);
[2m  File "[0m[1m[35m$DIR/src/lfortran/semantics/ast_common_visitor.h[39m[0m[2m", line 1086, in LFortran::CommonVisitor<LFortran::SymbolTableVisitor>::visit_DeclarationUtil(LFortran::AST::Declaration_t const&)[0m
    if( is_c_ptr(v, derived_type_name) ) {
[2m  File "[0m[1m[35m$DIR/src/lfortran/semantics/ast_common_visitor.h[39m[0m[2m", line 728, in LFortran::CommonVisitor<LFortran::SymbolTableVisitor>::is_c_ptr(LFortran::ASR::symbol_t*, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >)[0m
    ASR::symbol_t* v_orig = ASRUtils::symbol_get_past_external(v);
[2m  File "[0m[1m[35m$DIR/src/libasr/asr_utils.h[39m[0m[2m", line 43, in ??[0m
    if (f->type == ASR::symbolType::ExternalSymbol) {
[2m  Binary file "[0m[1m[35m/lib/x86_64-linux-gnu/libc.so.6[39m[0m[2m", in killpg()[0m
Segfault: Signal SIGSEGV (segmentation fault) received
